### PR TITLE
BOD Bribes/Kotl city

### PR DESCRIPTION
**Pull request by kevin-10**
BOD Bribes: https://uo.com/wiki/ultima-online-wiki/gameplay/crafting/bulk-orders/#bribe
Kotl City: https://uo.com/wiki/ultima-online-wiki/combat/pvm-player-versus-monster/treasures-of-the-kotl-city/

Treasures of Kotl City can be toggles on/off with the following commands:
- EnableTKC
- DisableTKC
- ToggleTKC

Added IRevealableItem for ease of handing detectable/revealable items, such as traps, hidden chests, etc.
Created base class and base gump for item turn in.  For now, Agent of the Crown and Hal the Kotl Artifact collecter will derive for arty turn in.  
BaseTurnInMobile will have ankh tithe and resurrect functionality as per EA.

### CODE REVIEW

**Comment by Argalep** [comment]
HouseCraftables.cs error : IFlippable change to IFlipable

File: Scripts/Items/Functional/HouseCraftables.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public enum CraftableAddonType
         EndCurledMetalSignHanger,
     }
 
-    public class CraftableHouseAddonComponent : AddonComponent, IFlippable
+    public class CraftableHouseAddonComponent : AddonComponent, IFlipable
     {
         public int NorthID { get { return 0; } }
         public int WestID { get { return 0; } }
```
Comment (kevin-10) [inline Scripts/Items/Functional/HouseCraftables.cs]: Its in there, this is in HouseCraftables.cs.

File: Scripts/Items/Internal/ItemInterfaces.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ public interface IOwnerRestricted
         string OwnerName { get; set; }
     }
 
-    public interface IFlippable
+    public interface IFlipable
     {
         int WestID { get; }
```
Comment (kevin-10) [inline Scripts/Items/Internal/ItemInterfaces.cs]: Here is the interface change.

**Comment by kevin-10** [comment]
Make sure you merged the entire thing.  Also, the travis thing has indicated that it compiles.

**Pull request merged**